tox.ini

```diff
@@ -70,6 +70,8 @@ deps =
     synchro37: tornado>=6,<7
     synchro37: nose
 
+setenv =
+    PYTHONWARNINGS="error,ignore:The distutils package is deprecated:DeprecationWarning"
```

Comment:
Do we still need to ignore this now that we use setuptools on >=3.10?

Reply:
The warning is still raised by [pip itself](https://github.com/pypa/pip/blob/c247ddce123513198858d4589c5ee198dcc6ba40/src/pip/_internal/locations/_distutils.py#L9)